scripts/install-mender.sh

```diff
@@ -328,6 +334,23 @@ EOF
             echo "  Success!"
         fi
     fi
+
+    # Setup for mender-gateway
+    if [[ "$SELECTED_COMPONENTS" == *"mender-gateway"* ]]; then
+        if [ "$DEMO" -eq 1 ]; then
+            echo "  Setting up mender-gateway with demo configuration, certificates and key"
+            url="$MENDER_COMMERCIAL_DOWNLOAD_URL$(printf ${MENDER_GATEWAY_EXAMPLES_URL_PATH_F} $VERSION $VERSION)"
+            curl -fLsS -H "Authorization: Bearer $JWT_TOKEN" -O "$url" ||
+                    (echo ERROR: Cannot get mender-gateway-examples from $url; exit 1)
+            extract_dir=$(mktemp -d)
+            tar -C $extract_dir -xf mender-gateway-examples-$VERSION.tar
+            rsync --links --recursive $extract_dir/mender-gateway-examples-$VERSION/* /
```

Comment:
* apart from double quotes around `${extract_dir}` I would also use `${VERSION}`
* this is about decompressing to / with trimming of the `mender-gateway-examples-$VERSION/` in leading paths, right? is so, then: `tar -C / --strip-components=1 -xf mender-gateway-examples-$VERSION.tar` (but please test it typing from half-memory)

Reply:
@merlin-northern I didn't know about `--strip-components=1`, thank you very much! :pray: